app/components/autocomplete.tsx

```diff
@@ -1,50 +1,29 @@
-import { isEmpty } from "lodash";
-import { IAccessibleAutocomplteProps, ISpecies } from "~/types";
+import Autocomplete from 'accessible-autocomplete/react';
+import { t } from 'i18next';
 
-export const AccessibleAutocomplete = ({
-  id,
-  name,
-  value,
-  defaultValue,
-  defaultSelectMessage,
-  nojsValues,
-  error,
-}: IAccessibleAutocomplteProps) => (
-  <div
-    className={`govuk-form-group ${
-      !isEmpty(error) ? "govuk-form-group--error" : ""
-    }`.trim()}
-  >
-    <label
+
+export const AccessibleAutocomplete = (props)=> {
+
+  const suggest = async (query: string, populateResults) => {
+    console.log
```

Comment:
remove please

Reply:
Sure. In the main repo this will be picked up by EsLint and enforced with Husky precommit